frontend/src/APIClients/LogRecordAPIClient.ts

```diff
@@ -12,7 +12,7 @@ import {
 } from "../types/LogRecordTypes";
 
 const countLogRecords = async ({
-  building = "",
+  buildingId,
```

Comment:
after the prev changes this can be `buildingId = []` (you'll have to change the param type)